### PR TITLE
Hello Microverse

### PR DESCRIPTION
**Pull request by bienvenuushindi**
## hello-microverse

**In this project, I did the following;**

- Added an h1 in the `index.html` file
- Styled the h1 in the `style.css` file
- Ignored the `test.md` and `node_modules` file 
- Ensured there was no linter error

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,85 @@
...
+> remove: Total 10 (delta 1), reused 10 (delta 1)
+> Unpacking objects: 100% (10/10), done.
+```
+
+
+###
+
+###
+
+
```
Comment (zieeco) [inline README.md:57]: **[REQUIRE CHANGES]**

- Please, let's remove the unused lines to make our readme look more professional

File: README.md [hunk 2]
```diff
@@ -1,2 +1,85 @@
...
+## Built With
+
+- HTML, CSS
+- Git, Linters
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
```
Comment (zieeco) [inline README.md:16]: **[REQUIRED CHANGES]**

- Please, let's ensure our live demo link takes the user to the correct link. Right now, the live demo link doesn't takes the user to the correct link. Please, let's fix this issue.

File: README.md (hunk 1 above)
Comment (zieeco) [reply README.md:57]: **[REQUIRE CHANGES]**


- Please, let's rename the PR title `Feature branch contains new page structure and design` to `Hello Microverse` as this is more descriptive than the former :pray: 
- Let's update our PR by adding a summary describing introduced changes in more detail to make our PR more descriptive. we can try something like:

```

## hello-microverse

**In this project, I did the following;**

- Added an h1 in the `index.html` file
- Styled the h1 in the `style.css` file
- Ignored the `test.md` and `node_modules` file 
- Ensured there was no linter error
```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @bienvenuushindi 


Great work on the hello microverse project 👏 
You've done well implementing the setup and the project, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- Good gitflow usage ✔️


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by bienvenuushindi** [comment]
Hello @zieeco  
Thanks for reviewing my code I appreciate your suggestions
Is everything alright now

**Review by danmainah — APPROVED** [review]
Hi @bienvenuushindi , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design
✔️  Good PR and Description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**